src/views/storagemigrations/list/components/StorageMigrationRow.tsx

```diff
@@ -46,10 +46,10 @@ const StorageMigrationRow: FC<StorageMigrationRowProps> = ({
         />
       </TableData>
       <TableData activeColumnIDs={activeColumnIDs} id="storagemigration">
-        {getSelectedPVFromMigPlan(obj).length} {t('Volumes')}
+        {getSelectedPVFromMigPlan(obj)?.length || '0'} {t('Volumes')}
```

Comment:
1. it feels a param for the translation i.e. `t('{{ count }} Volumes`, {count:  ...)`
2. the existing code has 2 different paths for zero

Reply:
the `getSelectedPVFromMigPlan` function retrieve all the pvcs that have 'copy' as a method and not 'skip' 
Someone from cli can create a migplan with PVC with all 'skip' and here we should at least show 0 not crash the page.